composer.lock

```diff
@@ -73,7 +73,7 @@
                 "docblock",
                 "parser"
             ],
-            "time": "2015-08-31 12:32:49"
+            "time": "2015-08-31T12:32:49+00:00"
```

Comment:
I would say yes, as it showcases best practices for projects, and it ensures that people cloning the demo gets working dependencies, which is a better experience for a first try